packages/create-ripple-app/tidemodules.js

```diff
@@ -1,17 +1,17 @@
 module.exports = [
   { value: 'site', name: 'Site', default: true },
+  { value: 'preview', name: 'Preview', default: true },
   { value: 'alert', name: 'Alert', default: true },
   { value: 'page', name: 'Page', default: true },
   { value: 'landingPage', name: 'Landing Page', default: true },
   { value: 'event', name: 'Event', default: true },
   { value: 'news', name: 'News', default: true },
   { value: 'grant', name: 'Grant', default: false },
-  { value: 'gtm', name: 'Google Tag Manager', default: true },
   { value: 'profile', name: 'Profile', default: true },
   { value: 'media', name: 'Media', default: true },
   { value: 'webform', name: 'Webform', default: true },
   { value: 'search', name: 'Search', default: true },
-  { value: 'publication', name: 'Publication', default: false },
+  { value: 'publication', name: 'Publication', default: true },
```

Comment:
Is this change required for the preview?

Reply:
No, it's irrelevant to preview. Just correct the default value for it when I edit this file. It's the create ripple app default settings. Publication should be enabled for all now.